doc/cookbook/source/examples/distance/chi_square.rst

```diff
@@ -0,0 +1,35 @@
+===================
+Chi Square Distance
+===================
+
+The Chi Square Distance for real valued features x and x' extends the concept of :math:`\chi^{2}` metric to negative values.
+This distance is calculated by the equation:
+
+.. math::
+
+    d(\bf{x},\bf{x'}) = \sum_{i=1}^{n}\frac{(x_{i}-x'_{i})^2}{|x_{i}|+|x'_{i}|} \quad \bf{x},\bf{x'} \in R^{n}
```

Comment:
n is undefined

Reply:
Should I add "for any natural number n" ?

Reply:
I think it should be fine for defining n if you just put that the features belong to R^n the first time you introduce them. By the way, for extra mathy happiness, use \mathbb with R :nerd_face: